src/components/web-form-renderer.vue

```diff
@@ -289,28 +236,21 @@ const handleSubmit = async (payload) => {
   font-size: 16px;
 }
 html, body {
-  background-color: var(--gray-200);
-  box-shadow: none;
+  &:has(.odk-form) {
```

Comment:
I think we probably need to add this selector to `:root` above as well so that the font size on `:root` isn't applied on all pages.